### PR TITLE
[Hub Generated] Review request for Microsoft.Capacity to add version 2019-04-01

### PR DESCRIPTION
**Pull request by corquiri**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/app/branch/corquiri/azure-rest-api-specs/dev-reservations-Microsoft.Capacity-2019-04-01).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by corquiri** [comment]
Model validation check says there is one error but when I click View issues, it does not show any error.

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5477 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5477/commits/2622d035d908b8451745319e92a50f1aae75dfc7

**Comment by jhendrixMSFT** [comment]
@corquiri in the future please separate reformatting the document and meaningful changes into separate commits.  https://github.com/Azure/azure-rest-api-specs/pull/5403/commits/b6983f0dd2db88d915d6756ca9a55c37dc2fee42 is very hard to reason about due to the mix of whitespace and content changes.
If you can summarize the impacted APIs in the PR it would be a big help.

**Comment by corquiri** [comment]
@jhendrixMSFT Understood. There are 2 main changes

1. /providers/Microsoft.Capacity/calculatePrice added along with example "CalculateReservationOrder.json"
2. /providers/Microsoft.Capacity/reservationOrders/{reservationOrderId} has addition method "put". Response example is the same as that of "get"

The rest of api's are unchanged, except I created new $ref called "ReservationQuantity"

File: specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json [hunk 1]
```diff
@@ -0,0 +1,1261 @@
...
+              "$ref" : "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Capacity/reservationOrders/{reservationOrderId}" : {
+      "put" : {
+        "summary" : "Purchase `ReservationOrder`",
+        "description" : "Purchase `ReservationOrder` and create resource under the specificed URI\n",
```
Comment (nschonni) [inline specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json]: specificed -> specified

**Comment by corquiri** [comment]
question - can we hold onto merge of this request until changes are complete on RP side? If this PR gets merged, should the API be available immediately?

**Comment by jhendrixMSFT** [comment]
@corquiri Still waiting for ARM to review your API.  Also yes we don't merge swagger changes until the REST endpoints are live.

**Comment by corquiri** [comment]
@jhendrixMSFT hello, could you please provide update on this PR? I have pushed more changes and this PR is still waiting for ARM feedback. Is there ETA for ARM feedback? It has been 2 weeks since last update.

**Review by KrisBash — APPROVED** [review]
LGTM. Based on comments, I reviewed:

- PUT /reservationOrders/{id}
- POST /calculatePrice
- $ref ReservationQuantity

**Comment by salameer** [comment]
Merging based on ARM and Joel approval and the service is conformed to be live

**Pull request merged**